plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -72,6 +74,17 @@
      */
     private WriteConsistencyLevel consistency = WriteConsistencyLevel.DEFAULT;
 
+    /**
+     * Initial delay after a rejection before retrying request. Defaults to 50ms. With the default maxRetries to total time to retry
+     * rejects is about 25 seconds per rejection.
+     */
+    private TimeValue retryBackoffInitialTime = timeValueMillis(50);
+
+    /**
+     * Total number of retries attempted for rejections. Defaults to 10.
+     */
+    private int maxRetries = 10;
```

Comment:
Okay, I'm not expecting people to pass `Integer.MAX_VALUE`, I'm expecting people to think "-1 usually means disabled" and pass it, which will then blow up in the `writeVInt`/`readVInt` since it's negative. Do we validate that it's not negative anywhere?